src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");
+
+    }

```

Comment:
Remember to use informative messages in methods. Instead of 'Excavator started working.', it could be more descriptive, indicating what the Excavator is doing specifically. The same applies to the stopWork method.